src/server/PostManager.js

```diff
@@ -1,6 +1,34 @@
+import db from '../server/database';
+import {KeyUtils, ENTITY} from '../server/key-utils';
 
 export default class PostManager {
```

Comment:
I intended that this function have to belong to PostManger cause its about post/get/put/remove of post object.
If we deal this functions in PostManager, we don't have to make 'addPost' in ItemManager and ImageManager.